lib/validators/movies/list.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
+  title: Joi.string().min(1).max(255).optional(),
```

Comment:
Since the column name is `name`, this payload param should be `name` to. This applies to the create validator as well.